java/arcs/core/analysis/InformationFlow.kt

```diff
@@ -693,8 +693,14 @@ private fun Predicate.labels(): List<InformationFlowLabel> = when (this) {
 
 /** Returns true if the [check] is satisfied by the labels computed for [particle]. */
 fun InformationFlow.AnalysisResult.verify(particle: Recipe.Particle, check: Check): Boolean {
+  val log = TaggedLog { "AnalysisResult.verify" }
```

Comment:
This is too much debug information.  Let us not keep these log.debug statements.

Reply:
Sure thing, just pushed a reversion.